lib/scrivener/validations.rb

```diff
@@ -130,9 +131,7 @@ def assert_numeric(att, error = [att, :not_numeric])
       end
     end
 
-    URL = /\A(http|https):\/\/([a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}|(2
-          5[0-5]|2[0-4]\d|[0-1]?\d?\d)(\.(25[0-5]|2[0-4]\d|[0-1]?\d?\d)){3}
-          |localhost)(:[0-9]{1,5})?(\/.*)?\z/ix
+    URL = URI::regexp(%w(http https))
 
     def assert_url(att, error = [att, :not_url])
```

Comment:
Might make sense to pass the schemes here:

``` ruby
def assert_url(att, schemes = %w(http https), error = [att, :not_url])
  if assert_present(att, error)
    assert_format(att, URI.regexp(schemes), error)
  end
end
```

This would allow people to use this helper to check other kind of URLs without extra work. Thoughts?